src/Tmds.Ssh/ChannelContextSendMessageExtensions.cs

```diff
@@ -28,8 +28,9 @@ byte      SSH_MSG_CHANNEL_FAILURE
             }
         }
 
-        public static ValueTask SendChannelDataMessageAsync(this ChannelContext context,  ReadOnlyMemory<byte> memory, CancellationToken ct)
+        public static ValueTask SendChannelDataMessageAsync(this ChannelContext context, ReadOnlyMemory<byte> memory, CancellationToken ct)
         {
+            // TODO Use SendChannelDataAsync because of a Window?
```

Comment:
Good catch. This is a bug!

Reply:
Can you move this method in the `ChannelContext` class and make it `private`?

Reply:
Not sure if I understand you, ChannelContext is abstract class, so I guess you mean moving into  SshChannelContext, but I had to get rid of the static qualifiers